vello/README.md

```diff
@@ -107,7 +108,7 @@ renderer
 surface_texture.present();
 ```
 
-See the [`examples/`](https://github.com/linebender/vello/tree/main/examples) folder to see how that code integrates with frameworks like winit and bevy.
+See the [`../examples`](https://github.com/linebender/vello/tree/main/examples) directory to see how that code integrates with frameworks like winit and bevy.
```

Comment:
I think this probably likely to be more/just as confusing - I'd suggest maybe:
```suggestion
The [`examples`](https://github.com/linebender/vello/tree/main/examples) folder in the repository shows that code integrated with frameworks like winit and bevy.
```

The specific issue is that this README is intended to be read on crates.io and docs.rs, with the GitHub use cases only peripherally

Reply:
I simplified it and removed the confusing dots.